source/api_documentation/transactions_api/_intro_for_affiliates.rst

```diff
@@ -0,0 +1,68 @@
+  :header-rows: 1
+  :class: parameters
+
+  * - Parameter
+    - Description
+
+  * - from=
+    - Starting date in user's time zone, in format YYYY-MM-DD. Example: 2011-06-01. Inclusive.
+
+  * - to=
+    - Ending date in user's time zone, in format YYYY-MM-DD. Example: 2011-06-07. Inclusive.
+
+  * - limit=
+    -  Max number of transactions to return at a time. Defaults to 1000. Limited to at most 4000.
+
+  * - transaction_id=
+    - A specific transaction. If transaction_id is specified and no transaction with that ID is found, nothing will be returned. Returns at most 1 transaction.
+
+  * - call_record_id=
+    - All transactions belonging to a specific call. If call_record_id is specified and no call with that ID is found, nothing will be returned. Returns all transactions belonging to the call record. **Note:** If transaction_id *and* call_record_id are passed, the specified transaction must belong to the specified call. If it does, the transaction will be returned. If it does not, nothing will be returned.
+
+  * - start_after_transaction_id=
+    - Transaction_id to start retrieving after. This should be the last value retrieved previously. Default (or empty string) means start at the oldest.
+
+  * - include_columns=
+    - comma-separated list of field names to be returned in the response for each transaction (if not specified, all available fields for the account will be returned, minus any columns in exclude_columns)
+
+  * - exclude_columns=
+    - comma-separated list of field names to be excluded in the response for each transaction
+
+  * - column_separator=
+    - [.csv format only] Separator between columns. Default is , for comma-separated values. (Can be set to any other separator like \| for pipe-separated values or %09 for tab-separated values.)
+
+  * - row_separator=
+    -  [.csv format only] Separator between lines. Defaults to %0A for \n (line feed). Use %0D%0A for \r\n (carriage return + line feed).
+
+  * - include_header=
+    - [.csv format only] 1 to include a header row; 0 to omit the header row. Default is 1.
+
+  * - force_quotes=
+    - [.csv format only] 1 to quote all CSV fields; 0 to only quote fields that contain separators. Default is 0.
+
+  * - transaction_type=
+    - Filters for the type of transaction. Valid inputs are Call, Sale, or Signal. Sale maps to the Reported Conversion type.
+
+In order to ensure that all transactions are returned when using the from= and to= date query parameters,
+you should store the last transaction id you have downloaded and pass it as the start_after_transaction_id to the next request.
+Typical usage on the polling interval is to repeatedly call the API until no rows are returned, meaning you have downloaded all transactions.
+Please note, the "to" and "from" date range parameters are both necessary, providing only one or the other will not filter the results.
+
+We provide a helpful constant that can be used in the include_columns and exclude_columns options:
```

Comment:
All of the information above this line is common between advertiser and network documentation.

If a change needs to be made to the above then it will require making the same change in 2 places: this file and the original _intro.rst file. That is unideal. We want to DRY up the above information so that we only have to change 1 file if the information needs to be changed/updated.

I believe my suggestion that we discussed would result in a much DRYer solution.

1. Keep the original _intro.rst file
2. Move the section from this line down into it's own file: `_affiliate_intro_constants.rst`
3. Move the same section from the _intro.rst file to a new file: `_intro_constants.rst`
4. Ensure the advertiser and network documentation specify `.. include:: _intro_constants.rst` after `.. include:: _intro.rst`
5. Ensure the `affiliate_user.rst` file specifies `.. include:: _affiliate_intro_constants.rst` after `.. include:: _intro.rst`

DRY == **D**o not **R**epeat **Y**ourself